config/main.py

```diff
@@ -108,6 +108,110 @@ def _get_option(ctx,args,incomplete):
+        click.secho("[ERROR] {} is not a Parent port. So, Breakout Mode is not available on this port".format(interface_name), fg='red')
+        return False
+
+    # Check whether target breakout mode is available for the user-selected interface or not
+    if target_brkout_mode not in breakout_file_input[interface_name]["breakout_modes"]:
+        click.secho('[ERROR] Target mode {} is not available for the port {}'. format(target_brkout_mode, interface_name), fg='red')
+        return False
+
+    # Get config db context
+    config_db = ctx.obj['config_db']
+    port_dict = config_db.get_table('PORT')
+
+    # Check whether there is any port in config db.
+    if not port_dict:
+        click.echo("port_dict is None!")
+        return False
+
+    # Check whether the  user-selected interface is part of  'port' table in config db.
+    if interface_name not in port_dict.keys():
+        click.secho("[ERROR] {} is not in port_dict".format(interface_name))
+        return False
+    click.echo("\nRunning Breakout Mode : {} \nTarget Breakout Mode : {}".format(cur_brkout_mode, target_brkout_mode))
+    if (cur_brkout_mode == target_brkout_mode):
+        click.secho("[WARNING] No action will be taken as current and desired Breakout Mode are same.", fg='magenta')
+        sys.exit(0)
+    return True
+
+def load_configMgmt(verbose):
+    """ Load config for the commands which are capable of change in config DB. """
+    try:
+        cm = configMgmt(debug=verbose)
+        return cm
+    except Exception as e:
+        raise Exception("Failed to load the config. Error: {}".str(e))
+
+
+def delete_Ports(final_delPorts,force_remove_dependencies, verbose):
+    """
+    Delete all ports.
+    del_ports: list of port names.
+    force_remove_dependencies: if false return dependecies, else delete dependencies.
+    Return:
+        WithOut Force: (xpath of dependecies, False) or (None, True)
+        With Force: (xpath of dependecies, False) or (None, True)
+    """
+    deps, ret = cm.deletePorts(delPorts=final_delPorts, force=force_remove_dependencies)
+    # No further action with no force and deps exist
+    if not force_remove_dependencies:
+        if ret == False and deps:
+            print("Dependecies Exist. No further action will be taken")
+            print("\n*** Printing dependecies ***\n {}".format(deps))
+            sys.exit(0)
+    if ret == False and deps:
+        print("[ERROR] Port Deletion failed!!! Opting Out")
+        raise click.Abort()
+
+    print("\n*** Ports have been deleted successfully ***\n")
+
+
+def add_Ports(final_addPorts, port_dict, load_predefined_config, verbose):
+    """
+    Add Ports and default config for ports to config DB, after validation of data tree.
+    add_ports : list of ports
+    port_dict : Config DB Json Part of all Ports same as PORT Table of Config DB.
+    load_predefined_config : If True, add default config as well.
+    return:
+        Sucess: True or Failure: False
+    """
+    try:
```

Comment:
Plz add 
**portJson = dict(); portJson['PORT'] = port_dict** 
here, i.e. Pass as PORT table in config. Thx.